### ISSUE TITLE
NTR: gamma-glutamyl transpeptidase low in leukocytes or fibroblasts (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8190]
New term request gamma-glutamyl transpeptidase low in leukocytes or fibroblasts (segal_070405223339):   UMLS: 1510494

**Comment by pnrobinson** [comment]
@MickeySegal
it seems this enzyme can be measured in various places, see
https://www.sciencedirect.com/science/article/pii/S000649712071089X?via%3Dihub
Given this is a rarely used test, I think it might be OK just to have a general Reduced gamma-glutamyl transpeptidase activity term.
Thoughts?

**Comment by MickeySegal** [comment]
The paper you cited is about https://omim.org/entry/230450.  The enzyme in this NTR is for https://omim.org/entry/231950.

**Comment by pnrobinson** [comment]
gamma-glutamyltransferase (GGT; [EC 2.3.2.2](https://enzyme.expasy.org/EC/2.3.2.2))

@MickeySegal
Adding new term: 
- ID: HP:6000578
- label: Reduced tissue gamma-glutamyltransferase activity
- parents: [Abnormal tissue enzyme activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:29483667
- def: Activity of gamma-glutamyltransferase in the tissues below the lower limit of normal. The activity can be measured in multiple tissues including leukocytes and cultured fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8190
- synonyms:
